fdbserver/ClusterController.actor.cpp

```diff
@@ -1570,6 +1570,10 @@ class ClusterControllerData {
 			return false;
 		}
 
+		// Because a configuration with fewer proxies or resolvers does not cause this function to fail,
+		// we need an extra check to determine if the total number of processes has been reduced.
+		// This is mainly helpful in avoiding situations where avoiding a degraded process
```

Comment:
```suggestion
		// This is mainly helpful in avoiding situations where killing a degraded process
```
Did you mean to use a word other than "avoiding" here?